changelogs/fragments/junos_acl_multi_address_port_acls.yaml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

Comment:
This should be replaced with the `major_changes`  tag, as argspec change would require changes in Playbooks.

Reply:
Ok! I wasn't sure if the issue reference should have been changed somehow as well. Changed just the tag for now.